test/assignment.coffee

```diff
@@ -234,7 +234,71 @@ test "destructuring assignment against an expression", ->
   [y, z] = if true then [a, b] else [b, a]
   eq a, y
   eq b, z
-
+  
+test "destructuring assignment with objects and splats: ES2015", ->
+  obj = {a:1, b:2, c:3, d:4, e:5}
+  throws (-> CoffeeScript.compile "{a, r..., s....} = x"), null, "multiple rest elements are disallowed"
```

Comment:
It works in Babel, but I meant that it did not compile with the [destructuring_object branch](https://github.com/GeoffreyBooth/coffeescript/tree/destructuring_object) 
Apologies for not making myself clear the first time.